frontend/src/app/features/details/SummaryForm.tsx

```diff
@@ -1,11 +1,14 @@
-import React, { FC, useState } from 'react';
+import React, { FC, useEffect, useState } from 'react';
 import {
   FormFieldType,
   IFormField,
 } from '../../components/input-controls/IFormField';
 import Form from '../../components/form/Form';
 import { Sites } from '../site/dto/Site';
 import './SummaryForm.css';
+import { useSelector } from 'react-redux';
+import { isUserPurchasedSnapshot } from './snapshot/SnapshotSlice';
```

Comment:
pls change `isUserPurchasedSnapshot` to `hasUserPurchasedSnapshot`

Reply:
Updated.